python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +762,154 @@ def run(self, inputDefects, camera):
         return pipeBase.Struct(
             mergedDefects=merged,
         )
+
+# Subclass the MergeDefects task to reduce the input dimensions
+# from ("instrument", "detector", "exposure") to
+# ("instrument", "detector").
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
+                                      dimensions=("instrument", "detector")):
+    inputDefects = cT.Input(
+        name="singleExpDefects",
```

Comment:
This name should be `cpBrightPartialDefectsCombined` as defined in the default pipeline, and then the pipeline doesn't need the override.  That is, the default shouldn't be useless and immediately overridden.  However, I don't know if I like the name.  Are these *bright* defects or are they defects from the *dark*.  E.g, should it be `cpPartialDefectsFromDarkCombined` or some combination of those words?

Reply:
The defects measured from darks are referred to bright or hot (above dark current threshold).  I'll use `cpPartialDefectsFromDarkCombined` and put it in the code, deleting it from the yaml file.